<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VkSettingsPage</class>
 <widget class="QWidget" name="VkSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vk.com</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/data.qrc">
    <normaloff>:/providers/vk.png</normaloff>:/providers/vk.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="account">
     <property name="title">
      <string>Account details</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="login_button">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="preferences">
     <property name="title">
      <string>Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Max global search results</string>
          </property>
          <property name="buddy">
           <cstring>maxGlobalSearch</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxGlobalSearch">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>3000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="love_button_is_add_to_mymusic">
        <property name="text">
         <string>Add songs to &quot;My Music&quot; when the &quot;Love&quot; button is clicked</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="groups_in_global_search">
        <property name="text">
         <string>Show groups in global search result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Caching</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="enable_caching">
        <property name="text">
         <string>Enable automatic caching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="cache_opts" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Cache path:</string>
             </property>
             <property name="buddy">
              <cstring>cache_dir</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cache_dir"/>
           </item>
           <item>
            <widget class="QToolButton" name="choose_path">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>File name pattern:</string>
             </property>
             <property name="buddy">
              <cstring>cache_filename</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cache_filename">
             <property name="text">
              <string notr="true">%artist - %title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="reset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
